### PR TITLE
docs (EE): Add FAQ to multitenancy article

### PR DESCRIPTION
**Pull request by bucanero**
fixes DOC-206

### CODE REVIEW

**Review by anand-chandrashekar — CHANGES_REQUESTED** [review]

File: content/enterprise-features/multitenancy.md [hunk 1]
```diff
@@ -25,6 +25,36 @@ The default namespace is called a `galaxy`. [Guardians of the Galaxy](#guardians
...
+    Dgraph uses [Access Control Lists]({{< relref "access-control-lists.md" >}}) (ACL) to manage controls and policies.
+    Previously, Dgraph had ACL across a single instance. With multi-tenancy, Dgraph's ACL controlling scope is by tenant.
+    Each tenant must be set up using ACL, and only users associated with this ACL will have access to that tenant.
+
+{{% notice "note" %}}
+Only super-admins ([Guardians of the galaxy](#guardians-of-the-galaxy)) have access across tenants.
+The super admin is used only for database admininstration operations, such as exporting data of all tenants. 
+{{% /notice %}}
+
+- What is the granularity of the access controls? Is it per graph? Per tenant only?
```
[the remark this answers is not in the document]
Comment (bucanero) [reply content/enterprise-features/multitenancy.md:41]: fixed

File: content/enterprise-features/multitenancy.md [hunk 2]
```diff
@@ -25,6 +25,36 @@ The default namespace is called a `galaxy`. [Guardians of the Galaxy](#guardians
 special access to create or delete namespaces and change passwords of
 users of other namespaces.
 
+## FAQ
+
+- How access controls and policies are handled among different tenants?
+
+    Dgraph uses [Access Control Lists]({{< relref "access-control-lists.md" >}}) (ACL) to manage controls and policies.
+    Previously, Dgraph had ACL across a single instance. With multi-tenancy, Dgraph's ACL controlling scope is by tenant.
```
[the remark this answers is not in the document]
Comment (bucanero) [reply content/enterprise-features/multitenancy.md:33]: fixed

File: content/enterprise-features/multitenancy.md [hunk 3]
```diff
@@ -25,6 +25,36 @@ The default namespace is called a `galaxy`. [Guardians of the Galaxy](#guardians
...
+{{% notice "note" %}}
+Only super-admins ([Guardians of the galaxy](#guardians-of-the-galaxy)) have access across tenants.
+The super admin is used only for database admininstration operations, such as exporting data of all tenants. 
+{{% /notice %}}
+
+- What is the granularity of the access controls? Is it per graph? Per tenant only?
+
+    The access controls are applied per tenant at a predicate level.
+    For example, the user `John Smith` belonging to the group `Data Approvers` may only have read-only access to predicates,
+    while user `Jane Doe`, who belongs to the group `Data Managers`, can be given access to modify predicates.
```
[the remark this answers is not in the document]
Comment (bucanero) [reply content/enterprise-features/multitenancy.md:45]: fixed (using `data editors`)

File: content/enterprise-features/multitenancy.md [hunk 4]
```diff
@@ -25,6 +25,36 @@ The default namespace is called a `galaxy`. [Guardians of the Galaxy](#guardians
...
+    All of these ACL constraints have to be configured for each tenant. 
+
+- Are tenants a physical separation or a logical one? Does the data needs to be writen twice (e.g., for 2 tenants), or only once?
```
[the remark this answers is not in the document]
Comment (bucanero) [reply content/enterprise-features/multitenancy.md:48]: removed

**Review by anand-chandrashekar — APPROVED** [review]

**Pull request merged**